frappe/__init__.py

```diff
@@ -356,6 +357,16 @@ def _raise_exception():
 
 		out.message = '''<table class="table table-bordered"
 			style="margin: 0;">{}</table>'''.format(table_rows)
+	
+	if as_list and type(msg) in (list, tuple):
+		if len(msg) > 1:
+			list_rows = ''
+			for row in msg:
+				list_rows += '<li>{}</li>'.format(row)
+
+			out.message = '''<ul style="padding-left: 20px">{}</ul>'''.format(list_rows)
+		elif len(msg) == 1:
+			out.message = msg[0]
```

Comment:
@nextchamp-saqib the html should be handled client side, in `messages.js`

Reply:
Done.